transcom/mymove
# Update cypress to 10.3.1

## Summary

We need to update Cypress to be up to date with CircleCI-Docker. At the moment they are on different versions and MyMove needs to catch up.

### Changes made for the upgrade:
* import plugins/index.js to cypress.config.js and use code in SetupNodeEvents() func
* replace `cypress/support/index.js` with `cypress/support/e2e.js`
* rename cypress/integration to cypress/e2e and  append all files in associated folders with `cy.js` instead of `.js`


## Verification Steps for Author

These are to be checked by the author.


## Verification Steps for Reviewers

Check to make sure none of our tests are breaking as a result of this dependency update.
